facebookads/api.py

```diff
@@ -143,10 +138,6 @@ class FacebookAdsApi(object):
 
     HTTP_METHOD_DELETE = 'DELETE'
 
-    HTTP_DEFAULT_HEADERS = {
-        'User-Agent': "fb-python-ads-api-sdk-%s" % SDK_VERSION,
-    }
```

Comment:
@ebzlo I had to do that because of cyclical import. It might break compatibility